Description: Don't try to use a Sphinx extension we don't have

Author: Rebecca N. Palmer <rebecca_palmer@zoho.com>
Forwarded: not-needed

--- a/doc/source/conf.py
+++ b/doc/source/conf.py
@@ -57,7 +57,6 @@ extensions = [
     "numpydoc",
     "sphinx_copybutton",
     "sphinx_design",
-    "sphinx_toggleprompt",
     "sphinx.ext.autodoc",
     "sphinx.ext.autosummary",
     "sphinx.ext.coverage",
